Increase timeout for tests, needed when running tests on GPU.

Åke Sandgren, 2019-01-03
Adapted for 2023.1 by Bob Dröge, 2023-05-11
diff -ru gromacs-2023.1.orig/src/testutils/TestMacros.cmake gromacs-2023.1/src/testutils/TestMacros.cmake > GROMACS-2023.1_increase_test_timeout_for_GPU.patch
--- gromacs-2023.1.orig/src/testutils/TestMacros.cmake	2023-04-21 15:12:08.000000000 +0200
+++ gromacs-2023.1/src/testutils/TestMacros.cmake	2023-05-11 15:57:16.625566730 +0200
@@ -274,7 +274,7 @@
             elseif (${CMAKE_BUILD_TYPE} STREQUAL TSAN)
                 set(_timeout 300)
             else()
-                set(_timeout 120)
+                set(_timeout 240)
             endif()
         elseif (ARG_SLOW_TEST)
             list(APPEND _labels SlowTest)
